src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,695 @@
+---
+title: 'SCIM API tutorial'
+tags:
+  - Accounts
+  - Accounts and billing
+  - Automated user management
+  - New Relic One user management
+metaDescription: 'New Relic SCIM API tutorial: automated user management of groups and users using the SCIM API.'
+---
+
+This tutorial will walk you through some common procedures for managing users on the New Relic One user model using the SCIM API. The SCIM API allows you to view, create, update, and delete users and groups programmatically, outside of the [User management UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where).
+
+## Requirements [#requirements]
+
+Before starting this tutorial, please read the [requirements for automated user management](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign) and for [the SCIM API](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management). 
+
+Note that the API does **not** give you the ability to manage access grants (assigning your New Relic users access to accounts and roles). This means that after you're done creating an integration with this API, you must next use the **Organization and access** UI to create access grants. For more information, see [user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
+
+## Overview [#overview]
+
+Before using this tutorial, we first recommend reading these docs:
```

Comment:
How about "we recommend you read"?

Reply:
I'd move this "to read" list up to the requirements section. Keep the overview a strict overview.